src/Messages/TSMessagesManager.m

```diff
@@ -1001,8 +1001,12 @@ - (void)processException:(NSException *)exception envelope:(OWSSignalServiceProt
       } else if ([exception.name isEqualToString:InvalidVersionException]) {
           errorMessage = [TSErrorMessage invalidVersionWithEnvelope:envelope withTransaction:transaction];
       } else if ([exception.name isEqualToString:UntrustedIdentityKeyException]) {
-          errorMessage =
-              [TSInvalidIdentityKeyReceivingErrorMessage untrustedKeyWithEnvelope:envelope withTransaction:transaction];
```

Comment:
I'll leave a note to the effect of: "We no longer create these, but legacy ones continue to exist"